src/_lib/modules/email-input/email-input.component.ts

```diff
@@ -83,7 +83,8 @@ export class EmailInputComponent implements OnInit, OnDestroy {
 
     onBlur() {
         this.isPillSelected = false;
-        this._addEmail();
+        // On blur, i.e when user clicks away, any unsubmitted text is discarded.
+        this.emailInputControl.setValue('');
```

Comment:
This is a suggestion. To me it's weird that my unfinished text is submitted when I click away, I'd rather it were discarded. If you disagree, I can remove this commit.

Reply:
So yeah, I'd remove this. I believe this is by design and the validity of the emails is dependent on the value of the input. I seem to remember discussing this with someone at the time